### ISSUE TITLE
TBC - Assist can not send sheets

### ISSUE THREAD

**Issue by nwardoh** [Ermad/angry-assignments#2]
Can it be made so that anyone with assist can send the assignments rather than just the raid leader?

**Comment by ywfn** [comment]
Should already be that way, assuming the receiver has the raid leader's name in his approved raid leaders list.  Or is this some bug that's specific to TBC only?

**Comment by nwardoh** [comment]
Can we use the "allow all"?

**Comment by ywfn** [comment]
Yeah, that should achieve the same thing.  As long as the raid leader is "approved", then any assistant in that raid is supposed to be able to set assignments.  (Approving a specific assistant doesn't achieve anything, the leader has to be approved to enable assistants in that raid to set assignments.)
